packages/runtime/container-runtime/src/summary/orderedClientElection.ts

```diff
@@ -416,13 +416,25 @@ export class OrderedClientElection
 			change = true;
 		}
 		if (change) {
+			this.logger.sendTelemetryEvent({
+				eventName: "SummarizerClientElected",
+				electedClientId: this._electedClient?.clientId,
```

Comment:
Maybe put all three props under `details` since we don't have any more columns in kusto? Also, it will be easier to find that way.

Reply:
yes, we already have these columns and we are not using them - it will be confusing to add a details for a column that is already there and will not be updated.